fe/fe-core/src/main/java/com/starrocks/catalog/Index.java

```diff
@@ -46,14 +47,18 @@
 import java.io.DataOutput;
 import java.io.IOException;
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
+import java.util.Map;
 import java.util.Objects;
 
 /**
  * Internal representation of index, including index type, name, columns and comments.
  * This class will used in olaptable
  */
 public class Index implements Writable {
+    @SerializedName(value = "indexId")
+    private long indexId;
```

Comment:
You introduced a indexId in this PR, I think this is a big change, because it can introduce some compatibility issue.

Reply:
Can you introduce this id later?

Reply:
Ok, I will revert them first and commit later.